elasticsearch/reindex/progress.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Usage: [watch -t] ./progress.sh <current index doc count>
+
+DOC_COUNT=$1
```

Comment:
Technically the old count is the count at the time you started the ingestion, right?